data/salesforce/src/main/java/org/teiid/spring/data/salesforce/SalesforceConfiguration.java

```diff
@@ -20,111 +20,128 @@
-    private Long requestTimeout = Long.MAX_VALUE;
-    private Long connectTimeout = Long.MAX_VALUE;
-    private Integer pollingInterval = 500;
-    private String clientId;
-    private String clientSecret;
-    private String username;
-    private String password;
-    private String refreshToken;
-    private RestTemplate restTemplate = new RestTemplate();
-
-    public String getUrl() {
-        return url;
-    }
-
-    public void setUrl(String url) {
-        this.url = url;
-    }
-
-    public SalesforceOAuth2Template getOAuth2Template() {
-        SalesforceOAuth2Template template = new SalesforceOAuth2Template(clientId, clientSecret,
-                "https://login.salesforce.com/services/oauth2/authorize",
-                "https://login.salesforce.com/services/oauth2/token");
-        template.setUseParametersForClientAuthentication(true);
-        return template;
-    }
-
-    @Override
-    public String getURL() {
-        return url;
-    }
-
-    @Override
-    public String getUsername() {
-        return username;
-    }
-
-    public void setUsername(String name) {
-        this.username = name;
-    }
-
-    @Override
-    public Long getRequestTimeout() {
-        return requestTimeout;
-    }
-
-    public void setRequestTimeout(Long requestTimeout) {
-        this.requestTimeout = requestTimeout;
-    }
-
-    @Override
-    public Long getConnectTimeout() {
-        return connectTimeout;
-    }
-
-    public void setConnectTimeout(Long connectTimeout) {
-        this.connectTimeout = connectTimeout;
-    }
-
-    public Integer getPollingInterval() {
-        return pollingInterval;
-    }
-
-    public void setPollingInterval(Integer pollingInterval) {
-        this.pollingInterval = pollingInterval;
-    }
-
-    public String getClientId() {
-        return clientId;
-    }
-
-    public void setClientId(String clientId) {
-        this.clientId = clientId;
-    }
-
-    public String getClientSecret() {
-        return clientSecret;
-    }
-
-    public void setClientSecret(String clientSecret) {
-        this.clientSecret = clientSecret;
-    }
-
-    @Override
-    public String getPassword() {
-        return password;
-    }
-
-    public void setPassword(String password) {
-        this.password = password;
-    }
-
-    public RestTemplate getRestTemplate() {
-        return restTemplate;
-    }
-
-    public void setRestTemplate(RestTemplate restTemplate) {
-        this.restTemplate = restTemplate;
-    }
-
-    public String getRefreshToken() {
-        return refreshToken;
-    }
-
-    public void setRefreshToken(String refreshToken) {
-        this.refreshToken = refreshToken;
-    }
+	private String url = "https://login.salesforce.com/services/Soap/u/45.0";
+	private String authorizeUrl = "https://login.salesforce.com/services/oauth2/authorize";
+	private String tokenUrl = "https://login.salesforce.com/services/oauth2/token";
```

Comment:
I think all this is going to be constant throughout the application. Can we make all the constant string as public, static and final?

Like *public static final String url = "something"*

Also, instead of defining the constant as camel case, define them in capital letters with underscore.

See: https://explainjava.com/java-constants/ for ref.

Reply:
1. I just used style of code from source project.
2. This is not just constants, it is properties with default values.